repl/pom.xml

```diff
@@ -174,6 +211,15 @@
                         <goals>
                             <goal>copy-dependencies</goal>
                         </goals>
+                        <configuration>
+                            <excludeArtifactIds>
+                                json4s-ast_${scala.binary.version},
+                                json4s-core_${scala.binary.version},
+                                json4s-jackson_${scala.binary.version},
+                                paranamer,
+                                scalap
```

Comment:
scalap will be assembled into shade jar, so don't need to copy into jars folder.